### PR TITLE
AB-117: Add functions for creating dataset dumps

### PR DESCRIPTION
**Pull request by ferbncode**
Added functions `dump_dataset_tables`  and `import_datasets_dumps` for exporting and importing dataset dumps. 

To create dumps, one can use:
`python manage.py dump full_dataset_dump`
To import dataset dump, one can use:
`python manage.py import_data --is-dataset-dump export/<dump.tar.xz>`

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -119,11 +119,16 @@ def init_test_db(force=False):
 
 @cli.command()
 @click.argument("archive", type=click.Path(exists=True))
-def import_data(archive):
+@click.option("--is-dataset-dump", "-d", is_flag=True, help="Import dataset dumps.")
```
Comment (paramsingh) [inline manage.py]: Would adding a new command for importing dataset dumps (like the new command for creating dataset dumps) make more sense?

File: db/dump.py [hunk 2]
```diff
@@ -176,64 +171,81 @@ def generate_where(row_name, start_t=start_time, end_t=end_time):
...
+                cursor.copy_to(f, 'dataset_class', columns=_DATASET_TABLES['dataset_class'])
+
+            # dataset_class_member
+            with open(os.path.join(location, "dataset_class_member"), "w") as f:
+                logging.info(" - Copying table dataset_class_member...")
+                cursor.copy_to(f, 'dataset_class_member', columns=_DATASET_TABLES['dataset_class_member'])
+
+        else:
+            # version
+            with open(os.path.join(location, "version"), "w") as f:
```
Comment (paramsingh) [inline db/dump.py]: Could we move these copy parts to a function called `copy_tables` and then call that for each table? Like [this](https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/db/dump.py#L293)

**Comment by ferbncode** [comment]
Updated the functions `dump_dataset_tables` and `import_datasets_dumps` for exporting and importing dataset dumps.

To create dumps, one can now use:
`./develop.sh run --rm webserver python manage.py dump full_dataset_dump`
To import dataset dump, one can use:
`./develop.sh run --rm webserver python2 manage.py import_dataset_data export/<dump.tar.xz>`

**Comment by paramsingh** [comment]
@ferbncode Please rebase against master once :)

**Comment by paramsingh** [comment]
This has become a part of #293, so closing.

**Pull request closed without merging**